### PR TITLE
Using expresssions

### PR DESCRIPTION
**Pull request by peteraba**
Just FYI, most static site generators will generate 'ids' for header tags (h1-h6), so you could probably just use `## Update Queries` instead of `<h2 id="update-queries">Update Queries</h2>`

### CODE REVIEW

File: database-query-builders.md [hunk 1]
```diff
@@ -174,6 +174,38 @@ The following values are bound to the query:
 
 > **Note:** `INSERT` and `UPDATE` query builders bind unnamed placeholder values.  To specify the type of the value, use an array whose first item is the value and whose second item is the type.
 
+<h3 id="using-expressions-in-insert-queries">Using Expressions In Insert Queries</h3>
+
+If you need to set some values as expressions, the column names to values array can contain `Expression` objects to help with that:
```
Comment (davidbyoung) [inline database-query-builders.md]:
```suggestion
If you need to set some values as expressions, you can use `Expression` objects in the column names to values array:
```

File: database-query-builders.md [hunk 2]
```diff
@@ -174,6 +174,38 @@ The following values are bound to the query:
...
+
+```php
+use Opulence\QueryBuilders\QueryBuilder;
+use Opulence\QueryBuilders\Expression;
```
Comment (davidbyoung) [inline database-query-builders.md]: Can you alphabetize the `use` statements?

File: database-query-builders.md [hunk 3]
```diff
@@ -206,6 +238,10 @@ The following values are bound to the query:
 
 > **Note:** Like `INSERT` query builders, `UPDATE` query builders bind unnamed placeholder values.
 
+<h3 id="using-expressions-in-update-queries">Using Expressions in Update Queries</h3>
+
+Using expressions to set value in `UPDATE` queries work exactly the same as for [INSERT Queries](#using-expressions-in-insert-queries).
```
Comment (davidbyoung) [inline database-query-builders.md]:
```suggestion
You can use SQL expressions in `UPDATE` queries just like you can with [`INSERT` queries](#using-expressions-in-insert-queries).
```

**Review by davidbyoung — COMMENTED** [review]
Can you also add links to the `<h3>` elements in the table of contents of this doc?

**Comment by davidbyoung** [comment]
Also, the reason I have to use manual IDs is because of weirdness around certain characters in Markdown titles.

**Comment by peteraba** [comment]
I pushed some fixes, please check them. Also, I noted your comment about the markdown titles. Never had such issues with hugo, but I could have also been just lucky I guess.

**Review by davidbyoung — APPROVED** [review]

**Pull request merged**